docs/vault/installation.md

```diff
@@ -4,10 +4,14 @@ Running a Vault will allow you to hold BTC of users in custody and in return ear
 
 To install the Vault client, follow this guide.
 
-At the end of this document you will have:
+## Checklist
 
-- [x] Started the Vault client
-- [x] Registered your Vault
+- [x] Generate Sr25519 key(s) for the parachain
```

Comment:
Please link to the right guides for the checklist so that vaults know where to find it in the docs.